### ISSUE TITLE
Externalise properties of deletion messages

### ISSUE THREAD

**Issue by joelcamus** [postwoman-io/postwoman#1078]
The message is static, I think that is a good idea, I send a language file. `en-US.json, es-ES.json ...`

![imagen](https://user-images.githubusercontent.com/8535654/90312708-dedc5180-df06-11ea-9688-670386763f5a.png)

```javascript
removeCollection() {
      if (!confirm("Are you sure you want to remove this Collection?")) return
      this.$store.commit("postwoman/removeCollection", {
        collectionIndex: this.collectionIndex,
      })
      this.syncCollections()
    },
```

The same happened in other collections, folders, requests and environments.

New developments:
- [x]  Collection
- [x]  Folder
- [x]  Request
- [x]  Environment

**Comment by liyasthomas** [comment]
Good observation. Hoping for a PR soon.

**Comment by joelcamus** [comment]
Done PR :ok_hand:

**Comment by liyasthomas** [comment]
Will get to it in a jiffy.

**Comment by liyasthomas** [comment]
Fixed in #1079
